torch_xla/experimental/torch_triton.py

```diff
@@ -0,0 +1,205 @@
+"""Module for calling Triton kernels from Pytorch/XLA."""
+
+from __future__ import annotations
+
+import os
+from typing import Any, Callable, Dict, Tuple, Union
+import zlib
+import torch
+
+import numpy as np
+import triton
+import triton.language as tl
+import torch_xla.csrc.triton._triton as lib_triton
+
+Grid = Union[int, Tuple[int], Tuple[int, int], Tuple[int, int, int]]
+GridOrLambda = Union[Grid, Callable[[Dict[str, Any]], Grid]]
+
+NUM_WARPS = 4
+NUM_STAGES = 3
+NUM_CTAS = 1
+
+
+def normalize_grid(grid: GridOrLambda, metaparams) -> Tuple[int, int, int]:
```

Comment:
It feels like a lot of helpers are actually copied from jax_triton: https://github.com/jax-ml/jax-triton/blob/ebdbde90c8129eb3e0471ee2d49f8281a9ed24f5/jax_triton/triton_lib.py#L74

Is that true?

Reply:
Yes, it is similar to the one used in jax-triton. Added a reference